### PR TITLE
feat: homepage

### PR DESCRIPTION
**Pull request by KeziahMoselle**

### CODE REVIEW

File: frontend/components/Hero.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export default function Hero({ title, children }) {
   return (
-    <section className="hero">
+    <section className="hero px-4">
```
Comment (stanriders) [inline frontend/components/Hero.js:3]: Is this file even being used? Isn't that a part of the starting template?

File: frontend/styles/globals.scss [hunk 2]
```diff
@@ -0,0 +1,128 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600&display=swap');
+
+@layer base {
+  @font-face {
+    font-family: 'Typo Round';
+    src: url('/fonts/Typo_Round_Regular_Demo.otf') format('opentype');
```
Comment (stanriders) [inline frontend/styles/globals.scss:10]: We really shouldn't use proprietary fonts here, I've asked Arhella to replace it with a free to use alternative

**Review by stanriders — APPROVED** [review]
Just a couple of comments that aren't blocking, but something to consider later

File: frontend/components/Hero.js (hunk 1 above)
Comment (KeziahMoselle) [reply frontend/components/Hero.js:3]: Yeah it's used in `src/index.js` but I don't think it needs to be a component yea

File: frontend/styles/globals.scss (hunk 2 above)
Comment (KeziahMoselle) [reply frontend/styles/globals.scss:10]: Yeah I had this question because this was a demo font but it was just to add one in the meantime :)

**Pull request merged**